lib/httplib/httplib_test.go

```diff
@@ -374,6 +375,67 @@ func TestSetIndexContentSecurityPolicy(t *testing.T) {
 			}
 		})
 	}
+
+	indexCSPStringCache = newCSPCache()
+}
+
+func TestSetIndexContentSecurityPolicy_OSSBuild(t *testing.T) {
+	modules.SetTestModules(t, &modules.TestModules{TestBuildType: modules.BuildOSS})
+
+	for _, tt := range []struct {
+		name            string
+		features        proto.Features
+		urlPath         string
+		expectedCspVals map[string]string
+		buildType       string
+	}{
+		{
+			name:     "default (no stripe or wasm)",
```

Comment:
I wonder if we should parameterize the tests with the buildType and join the tables into a single one. (It looks like you considered this too?)